plugin/coc-quick-lint-js/esbuild.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Is a build system really necessary? Let's write plain, un-transpiled JavaScript unless transpiling is required or very helpful for some reason.